src/app/services/mail/mail.service.ts

```diff
@@ -364,23 +364,23 @@ export class MailService {
    * @param args.form the form document to retrieve some email data from
    * @param args.submission the submission document to retrieve some email data from
    * @param args.attachments attachments to append to the email, if any
-   * @param args.jsonData the data to use in the data collation tool to be appended to the end of the email
+   * @param args.dataCollationData the data to use in the data collation tool to be appended to the end of the email
    * @param args.formData the form data to display to in the body in table form
    */
   sendSubmissionToAdmin = async ({
     replyToEmails,
     form,
     submission,
     attachments,
-    jsonData,
+    dataCollationData,
     formData,
```

Comment:
As we're encapsulating the complexity into the `EmailDataObj` class, could you combine the function arguments to accept an instance of that class instead? the service APIs should not need to know or ask for internal representations.